src/pass/vectorize_loop.cc

```diff
@@ -368,7 +368,7 @@ class Vectorizer : public IRMutator {
     CHECK(!op->extent.type().is_vector());
     Expr extent = Mutate(op->extent);
     if (extent.type().is_vector()) {
-      LOG(WARNING) << "Detect vectorized extent type, scalarizing...";
+      // LOG(WARNING) << "Detect vectorized extent type, scalarizing...";
```

Comment:
@merrymercy Do we need this warning?

Reply:
The reason we want to remove this warning is that AutoTVM may trigger it with some candidate configs in the tuning process and it looks annoying. If we really need to keep this warning, one alternative solution is trying to hide all warnings in AutoTVM, although I am not sure if it is doable since they are all managed by the same logging system.